compiler/hie-core/src/Development/IDE/LSP/Notifications.hs

```diff
@@ -32,7 +32,6 @@ setHandlersNotifications = PartialHandlers $ \WithMessage{..} x -> return x
     {LSP.didOpenTextDocumentNotificationHandler = withNotification (LSP.didOpenTextDocumentNotificationHandler x) $
         \_ ide (DidOpenTextDocumentParams TextDocumentItem{_uri,_version}) -> do
             updatePositionMapping ide (VersionedTextDocumentIdentifier _uri (Just _version)) (List [])
-            setSomethingModified ide
```

Comment:
This was just silly since `modifyFilesOfInterest` already takes care of kicking things off and kicking things off twice makes it annoying to find the right progress notification.